### PR TITLE
Updated the RenderListings function

### PR DESCRIPTION
**Pull request by Tokle89**
Rewrote the function to use createElements, instead of forcing a string into the DOM.
The function now also uses the API to get the data, instead of using a Lorem Ipsum, but still uses the same structure as before.
The function also uses the createElements function to create the elements, instead of creating them manually.
The Api call still comes from a dummy API, but the function is ready to be used with the actual API.
Deleted the searchListings import as it was never used

Now that the cards have different img and text, the styling needs to be adjusted, so the cards look alike

### CODE REVIEW

File: src/js/templates/listings/renderListings.js [hunk 1]
```diff
@@ -1,38 +1,104 @@
...
-               <span class="text-nowrap">(4) Applications</span>
-               <span>dd.mm.yy</span>
-               <a href="#" class="bg-theme-primary text-theme-black px-3 text-decoration-none" style="font-size: 1rem">View</a>
-             </div>
-           </div>
-         </div>
-       </div>
-       `;
+  const { products } = await getListOfListings();
+  console.log(products);
```
Comment (Olekrr) [inline src/js/templates/listings/renderListings.js:17]: We should generally avoid having console.logs in our final product, as frequent logging can impact performance or inadvertently expose sensitive information

**Review by Olekrr — CHANGES_REQUESTED** [review]
Code looks good to me, but the console.log should be removed

Well done!

File: src/js/templates/listings/renderListings.js (hunk 1 above)
Comment (Tokle89) [reply src/js/templates/listings/renderListings.js:17]: i Will fix this now, also discovered a problem on the button link i will fix at the same time

**Comment by Tokle89** [comment]
I have removed the console logs

changed the dummy api url to the noroff api url

created a parseDate function that converts the deadline date to readable format

**Comment by Olekrr** [comment]
Great, merging this :)

**Pull request merged**